composeApp/src/commonMain/kotlin/com/github/familyvault/ui/components/FamilyMemberEntry.kt

```diff
@@ -49,17 +51,19 @@ fun FamilyMemberEntry(
     var showDialog by remember { mutableStateOf(false) }
 
     Row(
-        modifier = Modifier.height(45.dp).fillMaxWidth(),
+        modifier = Modifier.height(45.dp).fillMaxWidth().clickable {
+            navigator.push(ModifyFamilyMemberScreen(familyMember.fullname))
```

Comment:
Niestety ale nie jest to oczywiste że aby zmienić grupę rodzinną trzeba kliknąc w kafelek. Może dodaj tutaj ikone trzech kropek itp zamiast znaku kosza i usunięcie z grupy rodzinnej wydziel do tego nowego ekranu modyfikacji